docs/modules/user-guide/partials/proc_defining-kubernetes-resources.adoc

```diff
@@ -100,26 +98,3 @@ items:
 ----
 
 For a basic example of a devfile with an associated Kubernetes or OpenShift list, see link:https://github.com/redhat-developer/devfile/tree/master/samples/web-nodejs-with-db-sample[web-nodejs-with-db-sample] on redhat-developer GitHub.
```

Comment:
This is a devfile v1 example but we dont have a devfile v2 sample, so I think we can keep it